libs/gltfio/src/AssetLoader.cpp

```diff
@@ -1012,9 +1020,17 @@ MaterialInstance* FAssetLoader::createMaterialInstance(const cgltf_material* inp
             break;
     }
 
+    // Check if this material has an extras string.
+    CString extras;
+    const cgltf_size extras_size = inputMat->extras.end_offset - inputMat->extras.start_offset;
+    if (extras_size > 0) {
```

Comment:
@pixelflinger @prideout this one would be as optimal as it can get using your approach to extras.